internal/usecase/handlers.go

```diff
@@ -0,0 +1,53 @@
+package usecase
+
+import (
+	r "address-book/internal/usecase/repository"
+
+	"github.com/gin-gonic/gin"
+	"net/http"
+)
+
+const (
+	NotFoundResponse = "Address by this param not found"
+	StatusOKResponse = "OK"
+)
+
+type Handler struct{}
+
+func (h Handler) CreateField(c *gin.Context, repo *r.Repository) {
+	repo.AddField(
+		c.PostForm("name"),
+		c.PostForm("address"),
+		c.PostForm("phone"),
```

Comment:
It seems, `name`, `address` and `phone` param names are used more than once. In that case, it's better to use consts or vars, instead of hard-coded values